dcos/model_metadata.go

```diff
@@ -0,0 +1,16 @@
+/*
+ * DC/OS
+ *
+ * DC/OS API
+ *
+ * API version: 1.0.0
+ */
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package dcos
+
+type Metadata struct {
+	PUBLICIPV4 string `json:"PUBLIC_IPV4,omitempty"`
```

Comment:
Could we get `PUBLIC_IPV4` instead?

Reply:
not sure how to enforce. Although better would be something like `PublicIPV4` but not sure how to influence the struct from OpenAPI description. Any suggestions?

Reply:
In the spec we have `x-go-name` for one property, but it seems to be coming from the go-swagger project, I don't see anything similar in `openapi-generator`. Currently there are other places where the naming is not purely Go idiomatic (see https://github.com/OpenAPITools/openapi-generator/issues/2740). 

Even though that were supported, I'm not a big fan of mixing the spec with a bunch of vendor extensions which are related to the Go client code generation, it seems like an anti-pattern. If we want to have better naming we should rather patch the tool generating the Go code to support more acronyms, uppercased properties, etc. In the meanwhile I think we should accept that naming isn't always 100% optimal.